backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -17,19 +17,22 @@
   @NotNull private String workPhoneNumber;
   private String cellPhoneNumber;
   @NotNull private String mailingAddress1;
-  private String aptSuiteOther;
-  private String aptFloorSuiteNo;
+  private String streetAddress2;
```

Comment:
Is there a reason we use `mailingAddress1` and `streetAddress2`?  It seems like we should settle on either mailing or street.  These aren't stored, but if they were would probably use street.

Reply:
No particular reason - I just wasn't sure if it used mailingAddress1 for consistency elsewhere. Changed to use streetAddress1 and streetAddress2 instead.